### PR TITLE
Eslint tweaks

### PR DESCRIPTION
**Pull request by dannon**
## What did you do? 
- Upgrade eslint/eslint-plugin-vue
- Swap attribute hyphenation (strongly recommended in vue) to enforced.  We definitely want to do this.
- Add curly enforcement for consistency.  I'm 80/20 on this one, but for consistency and readability is probably worth doing?  Mainly opening for discussion, I can drop those changes if we don't want to do it.

## Why did you make this change?
xref comments in #11701 re: readability of single-line conditionals being mixed with multi -- explicit curly-braces used consistently are easier to read.

## How to test the changes? 
(select the most appropriate option; if the latter, provide steps for testing below)
- [x] This is a refactoring of components with existing test coverage.

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Review by OlegZharkov — COMMENTED** [review]
From my point of view, 'if without curly brackets' is good enough as long as it's within a single line. It makes our code more concise. But I don't have an extremely strong opinion on this.

File: client/src/components/History/Scroller.vue [hunk 1]
```diff
@@ -199,7 +199,9 @@ export default {
         // determines cursor from scrollTop, need to wait until after render to calculate
         // this stuff because it depends on DOM elements
         updateCursor() {
-            if (this.suppressEvents) return;
+            if (this.suppressEvents) {
```
Comment (OlegZharkov) [inline client/src/components/History/Scroller.vue:202]: wow, it even changed comments

**Comment by dannon** [comment]
> From my point of view, 'if without curly brackets' is good enough as long as it's within a single line. It makes our code more concise. But I don't have an extremely strong opinion on this.

It's slightly (two...) fewer characters to leave them out for one-liners, but I think consistently using the explicit form is better practice, more clear, and others have said it's easier to read, especially with any compound logic in the conditional.

**Comment by OlegZharkov** [comment]
> > From my point of view, 'if without curly brackets' is good enough as long as it's within a single line. It makes our code more concise. But I don't have an extremely strong opinion on this.
> 
> It's slightly (two...) fewer characters to leave them out for one-liners, but I think consistently using the explicit form is better practice, more clear, and others have said it's easier to read, especially with any compound logic in the conditional.

well, if you think it's the right way, then do it ;)

**Review by mvdbeek — APPROVED** [review]

**Comment by OlegZharkov** [comment]
Looks like prettier doesn't fix one-line parenthesis by itself :cry:

**Pull request merged**